js/about/contributionStatement.js

```diff
@@ -430,4 +455,242 @@ function longFormattedDateFromTimestamp (timestamp) {
   return `${momentDate.format('MMMM Do')} at ${momentDate.format('h:mma')}`
 }
 
+const containerMargin = '25px'
+const boxMargin = '15px'
+const summaryBoxTableBorder = '10px'
+const summaryBoxTableMargin = '10px'
+const summaryBoxTablePadding = '10px'
+const lightGray = '#f7f7f7'
+const braveLogoWidth = '50px'
+const braveLogoHeight = '50px'
+const braveLogo = require('../../app/extensions/brave/img/braveAbout.png')
+const verifiedIcon = require('../../app/extensions/brave/img/ledger/verified_green_icon.svg')
+
+const styles = StyleSheet.create({
```

Comment:
One suggestion maybe. Instead of having one big object `styles`, maybe create 4 objects:

- styles (general and small styles)
- stylesHeader
- stylesPage
- stylesTable

What do you think?